external_review/external_review_collate.py

```diff
@@ -29,9 +28,6 @@ def collate(geography_level, category):
             )
             raise e
     final_df.index.rename(geography_level, inplace=True)
-    folder_path = f".staging/{category}"
```

Comment:
I see what you meant now. I am worried that if taking this out from this script would cause issue for categories don't have a meta data dump because they don't use `read_s3` at all?

Reply:
Oh yea good point. Then it should stay in both places?